content/rooms/fragments/v2-state-res.md

```diff
@@ -107,17 +107,18 @@ Now compare these two lists as follows.
 * If such a *j* exists, then *e<sub>j</sub>* = *P<sub>i</sub>* for some unique
   index *i* ≥ 0. Otherwise set *i* = ∞, where ∞ is a sentinel value greater
   than any integer.
-* In both cases, the *position of e in the mainline of P* is *i*.
+* In both cases, the *mainline position* of *e* is *i*.
 
 The *mainline ordering based on* *P* of a set of events is the ordering,
```

Comment:
now it's not obvious how P is involved in the definition below :(. Sorry, maybe my simplification wasn't a great idea, but otherwise we could do:

```suggestion
Given mainline positions calculated from *P*, the *mainline ordering based on* *P* of a set of events is the ordering,
```

Reply:
I definitely wasn't happy with the clunkiness of "the position of ____ in the mainline of ___" as a phrase. I think this is a good idea.